src/main/resources/static/external-merge-check-hook.soy

```diff
@@ -61,4 +61,26 @@
             ]
         ]/}
     {/call}
+	
+	{call aui.form.checkboxField}
+        {param legendContent: 'Cache Check Results' /}
+        {param fields: [
+            [
+                'id': 'cache_pr_results',
+                'labelText': 'Cache results of pull request check',
+                'isChecked': $config ? $config['cache_pr_results'] : true
+            ]
+        ]/}
+    {/call}
+	
+	{call aui.form.checkboxField}
+        {param legendContent: 'Include Details In Summary' /}
+        {param fields: [
+            [
+                'id': 'tooltip_results',
```

Comment:
Can you rename it to `detailed_tooltip`?

Reply:
Renamed.